### ISSUE TITLE
Create scrollable pane nunjucks macro

### ISSUE THREAD

**Issue by chrispymm** [ministryofjustice/moj-frontend#2112]
## What

Create a nunjucks macro for the scrollable pane component.

## Why

The scrollable pane component currently doesn't have a nunjucks macro - this is because it was just a div with a class.  There are extra attributes needed to ensure the component is accessible, so we should upgrade it to have a nunjucks macro to ensure these attributes are present.

## Who needs to be involved

- Developer

## Done

When the nunjucks macro exists.
